Mathlib/Algebra/Group/Defs.lean

```diff
@@ -647,6 +647,10 @@ lemma pow_mul' (a : M) (m n : ℕ) : a ^ (m * n) = (a ^ n) ^ m := by rw [Nat.mul
 lemma pow_right_comm (a : M) (m n : ℕ) : (a ^ m) ^ n = (a ^ n) ^ m := by
   rw [← pow_mul, Nat.mul_comm, pow_mul]
 
+@[to_additive ns_mul_congr]
+lemma pow_congr {a b : M} (n : ℕ) (h : a = b) : a ^ n = b ^ n :=
+  congrFun (congrArg HPow.hPow h) n
```

Comment:
Please drop this lemma. Just `rw [h]` whenever you want to apply it.

Reply:
Okay.